styleguide/source/assets/scss/00-base/_typography.scss

```diff
@@ -65,6 +65,8 @@ h4,
   }
 
   &--white {
+    @include font-size($h4-font-sizes--purple);
```

Comment:
I think the font modifications here and throughout are causing some visual regressions in backstop. Please run `backstop test` review the changes and update screenshots as necessary.

![_users_laurenbyrwa_sites_ama-style-guide-2_styleguide_backstop_data_html_report_index html 1](https://user-images.githubusercontent.com/1653723/41860412-42b6ddb2-7864-11e8-9836-0c64c88a767e.png)